examples/connector/golang/golang_connector/main.go

```diff
@@ -258,6 +270,16 @@ func (s *server) Test(ctx context.Context, in *pb.TestRequest) (*pb.TestResponse
 	}, nil
 }
 
+func LogMessage(level string, message string) {
```

Comment:
Should this be `logMessage` ? Or naming is fine?

Reply:
The method name starts with Capital letter in go, so this should be fine.

Reply:
Just searched about this:
In Go, method names typically start with a capital letter if you want them to be exported (accessible) from other packages. If the name starts with a lowercase letter, it will be unexported (private to the package).